File: 002+003/003.py
#nhap tam duong tron
#nhap ban kinh
#nhap toa do M
#kiem tra |MC| voi R

from vector import do_dai_vecto

xC, yC = map(float, input("Nhap toa do tam C(xC, yC)? ").split())
R = float(input("Nhap ban kinh R? "))
xM, yM = map(float, input("Nhap toa do M(xM, yM)? ").split())

l = float(do_dai_vecto(xC, yC, xM, yM))

if l < R:
    print("M nam trong C()")
elif l == R:
    print("M nam tren C()")
else:
    print("M nam ngoai C()")

File: 002+003/003.rb
#nhap tam duong tron
#nhap ban kinh
#nhap toa do M
#kiem tra |MC| voi R

require_relative "vector"

print "Nhap toa do tam C(xC, yC)? "
xc,yc = gets.split.map(&:to_f)
print "Nhap ban kinh R? "
r = gets.to_f
print "Nhap toa do M(xM, yM)? "
xm,ym = gets.split.map(&:to_f)

l = do_dai_vecto(xc,yc,xm,ym)

if l < r
  puts "M nam trong C()"
elsif l == r
  puts "M nam tren C()"
else
  puts "M nam ngoai C()"
end
